internal/setup.py

```diff
@@ -0,0 +1,21 @@
+from setuptools import setup
+
+setup(
+    name="AWSCloudFormationRPDK",
+    version="1.0",
+    packages=["rpdk.core"],
+    package_dir={"": "src"},
+    include_package_data=True,
+    entry_points={
+        "console_scripts": ["cfn-cli = rpdk.core.cli:main", "cfn = rpdk.core.cli:main"]
+    },
+    # Not all tests passed in Brazil due to some test dependency versions missing in Brazil
+    # Since all test are run in GitHub/CodeBuild, only verify one test
```

Comment:
Should this file be public? Also think test dependencies should be imported into Brazil rather than just disabling tests that depend on them